### PR TITLE
perf: support chinese counter

### PR DESCRIPTION
**Pull request by aleen42**

### CODE REVIEW

File: packages/counter-style/README.md [hunk 1]
```diff
@@ -151,6 +151,5 @@ All renderers can be chained to create variants, such as `withSuffix`,
 ### Limitations
 
 - `speakAs` hasn't been implemented yet.
-- [Chinese renderers](https://www.w3.org/TR/css-counter-styles-3/#limited-chinese)
   haven't been implemented, requiring custom logic. PRs welcomed!
```
Comment (jsamr) [inline packages/counter-style/README.md]: Typo: delete the two lines

**Comment by aleen42** [comment]
I have found that the native list item has only supported up to the integer limit `2147483647`. We can ensure the counter value within the range `[-2147483647, 2147483647]`.

```html
<ol start="2147483647" style="list-style-type: simp-chinese-informal;list-style-position: inside;">
    <li>1</li>
    <li>2</li> <!-- Cannot increase the count anymore -->
</ol>
```

**Comment by jsamr** [comment]
@aleen42 Great catch! Feel free to offer a distinct PR to cover this. Also, appreciate this PR, will review soon.

**Comment by aleen42** [comment]
> @aleen42 Great catch! Feel free to offer a distinct PR to cover this. Also, appreciate this PR, will review it soon.

I have implemented it and feel free to review it.

**Review by jsamr — APPROVED** [review]

**Pull request merged**